### ISSUE TITLE
AT128 mapping

### ISSUE THREAD

**Issue by pedestrain123** [vectr-ucla/direct_lidar_odometry#52]
Hello, may I ask if this algorithm supports Hesai semi-solid laser radar AT128？

**Comment by yaodix** [comment]
hello， here u again，have u test it with AT128？

**Comment by Nicky0325** [comment]
Yes, this algorithm works well with the AT128. If you'd like to try it without a ROS setup, I have a fork that runs directly on .pcd scans — no ROS dependency required:
👉 https://github.com/Nicky0325/direct_lidar_odometry
